eos-metapackage

```diff
@@ -86,6 +86,7 @@ sub subst_depends {
 foreach my $package (@{$dh{DOPACKAGES}}) {
 	subst_depends($package, "depends", "eos:Depends");
 	subst_depends($package, "recommends", "eos:Recommends");
+	subst_depends($package, "suggests", "eos:Suggests");
```

Comment:
Can you put this change in a separate commit? Probably doesn't matter, but it's independent and should stay even if we have to revert the rest of this for some reason.